app/src/main/res/layout/table_view_corner_layout.xml

```diff
@@ -1,16 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="@color/table_view_default_separator_color"
     android:minHeight="@dimen/default_cell_height">
 
     <ImageView
+        android:id="@+id/buttonRowScaleMinus"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_centerHorizontal="true"
+        android:tint="#333333"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/buttonRowScaleAdd"
+        app:srcCompat="@drawable/ic_scale" />
+
+   <!-- <ImageView
         android:id="@+id/buttonScale"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_centerInParent="true"
-        app:srcCompat="@drawable/ic_zoomx2" />
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/buttonRowScaleAdd"
+        app:layout_constraintStart_toEndOf="@id/buttonRowScaleMinus"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_zoomx2" />-->
```

Comment:
Are we going to need this in the near future??

Reply:
We are talking about this right now. It seems nope we won't.
I'll update this layout once the icons are decided.